cmd/network/setup_linux.go

```diff
@@ -99,6 +92,82 @@ func main() {
 		logrus.Fatal(err)
 	}
 
+	connFile, err := vsockConn.File()
+	if err != nil {
+		logrus.Fatal(err)
```

Comment:
All the `logrus.Fatal(err)` makes me suspect this would be clearer as a inner `func() error` and have `main()` just call it and `logrus.Fatal` if it's not `nil`… But probably not this PR.